experiments/dataSources/openAI/chatGPT_Demo/Readme.MD

```diff
@@ -0,0 +1,28 @@
+# Integrate a Chatbot into Your Apps
+<img src="../OpenAI_Introduction.png" width="500"></img>
+
+This page provides a demo Chatbot experience that you can integrate elsewhere in your app.
+
+## Instructions
+- Page API:  V2
+- Data source: 
+    REST data source named "OpenAI" connecting to OpenAI APIs per [instructions](openAI)
```

Comment:
I think this link needs correction so it goes up a folder..

Reply:
Can you give me some guidance here on how to correct it? I tried to find instructions online and can't figure it out. I tried to emulate the links in one of the other openAI projects and they're broken too 🙈